index.js

```diff
@@ -1,31 +1,147 @@
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio/',
+  },
+  ];
+
+  const gridTemplate = document.getElementById('grid-template');
+  const recentWorks = document.getElementById('Recent-id');
+  const detailsPopUpHeading = document.querySelector('.detail-popup-heading');
+  const popupWindow = document.getElementById('mobile-details-modal');
+  projects.forEach((project) => {
+    const clonedNode = gridTemplate.children[0].cloneNode(true);
+    clonedNode.children[0].children[0].innerText = project.name;
+    recentWorks.appendChild(clonedNode);
+
+    detailsPopUpHeading.textContent = project.name;
+
+    const btnPopups = recentWorks.querySelectorAll('.btn-popup');
+    btnPopups.forEach((button) => {
+      button.addEventListener('click', () => {
+        popupWindow.classList.remove('hidden');
+      });
+    });
+  });
```

Comment:
Kindly add the scroll for the details modal in mobile version width `375px` in order all your modal content appear.
![image](https://user-images.githubusercontent.com/24703171/157516954-3acddadc-5ec6-4c38-8849-cfe4bdf9b725.png)